client/blocks/inline-help/inline-help-search-results.jsx

```diff
@@ -146,6 +147,8 @@ function HelpSearchResults( {
 			'is-selected': selectedResultIndex === resultIndex,
 		} );
 
+		const external = externalLinks && support_type === SUPPORT_TYPE_CONTEXTUAL_HELP;
```

Comment:
I think the correct test here is that the result _isn't_ the admin section type.

```suggestion
		const external = externalLinks && support_type !== SUPPORT_TYPE_ADMIN_SECTION;
```

The current test works when the user hasn't searched for anything, but breaks if you enter any search query because in this case `support_type === SUPPORT_TYPE_API_HELP`